File: Score of a String/ScoreOfAString.dart
class Solution {
  int scoreOfString(String s) {
    int sum = 0;
    for (int i = 1; i < s.length; i++) {
      sum += (s[i].codeUnitAt(0) - s[i - 1].codeUnitAt(0)).abs();
    }
    return sum;
  }
}

File: Score of a String/ScoreOfAString.rs
impl Solution {
    pub fn score_of_string(s: String) -> i32 {
        let mut sum: i32 = 0;
        let bytes = s.as_bytes();
        for i in 1..bytes.len() {
            sum += (bytes[i] as i32 - bytes[i - 1] as i32).abs();
        }
        return sum;
    }
}
